src/utils/errorHandler.js

```diff
@@ -78,6 +78,9 @@ function handleError(error, customMessage, analyticsInstance) {
   }
 
   logger.error(errorMessage);
+  if(error.message && error.message.includes(FAILED_REQUEST_ERR_MSG_PREFIX)){
+    return; // do not notify errors in case the request has failed
```

Comment:
can we make FAILED_REQUEST_ERR_MSG_PREFIX on array of messages to filter out like [FAILED_REQUEST_ERR_MSG_PREFIX] and iterate through all so we can filter more cases in the future as they come up?

Reply:
@MoumitaM, do we have to handle the error in the XHR case? Won't it be sufficient to log the error message there itself?

Reply:
Yes, logging the error is sufficient. Now handleError method will do the same thing, only log the error.

Reply:
No, I mean, we don't have to take it to the handleError method to just log the error.

Reply:
Addressed